src/CardDeck.php

```diff
@@ -103,6 +103,7 @@ protected function init(): void
 
         if ($this->paginator !== false) {
             $this->addPaginator();
+            $this->stickyGet($this->paginator->name);
```

Comment:
if removed, does Behat testing fails?

Reply:
there is no behat test for Card deck with either VpExecutor or new PanelExecutor. This is because the default executor register in ExecutorFactory  is ModalExecutor. I can added tests but we need a way to be able to switch executor types dynamically. Any idea? Should we switch base on GET argument in init-app?

Reply:
I meant if sticky args in Card Deck are tested.

> with either VpExecutor or new PanelExecutor...

I belive this does not need extra cross testing